src/main/kotlin/com/sourcegraph/cody/agent/protocol/GetFeatureFlag.kt

```diff
@@ -1,3 +1,7 @@
 package com.sourcegraph.cody.agent.protocol
 
-data class GetFeatureFlag(val flagName: String)
+data class GetFeatureFlag(val flagName: String) {
```

Comment:
I am preparing us to have two more ff here (#319). I tried to make it an enum but it did not work through the request to the agent.